CTP/sql/bin/run.sh

```diff
@@ -767,66 +782,23 @@ function parse_command_from_core()
 
 }
 
-function search_core_and_backup()
+function check_core()
 {
      curDir=`pwd`
-     resultDir=$1
-     hasCore=`find $CUBRID ${CTP_HOME} -type f -name "core*"|wc -l`
-     if [ $hasCore -ne 0 ];then
-        echo "========================"
-        echo "COREDUMP file(s) exist!"
-        echo "========================"
-        if [ ! -d "$HOME/corebackup" ];then
-                mkdir $HOME/corebackup
-        fi
-        
-        cd $HOME/corebackup
-        timestamp=`date +'%y%m%d%H%M%S'`
-        coreFolder="${scenario_alias}_${cubrid_ver}_${timestamp}"
-        mkdir -p "${coreFolder}"
-        coreDir=${HOME}/corebackup/${coreFolder}
-        cd $coreFolder     
-       
-        coreFiles=`find $CUBRID ${CTP_HOME} -type f -name "core*"` 
-        for fcore in ${coreFiles};do
-            echo "print backtrace with gdb and $fcore..."
-            binary_file=`file ${fcore} | sed "s|.*from '\(.*\)'|\1|"|awk '{print $1}'`
-            binary_cmd=`parse_command_from_core ${binary_file}`
-           
-            cp $fcore .
-            
-            coreName=`basename $fcore`
-            coreTimeStamp=`date +'%y%m%d%H%M%S'`
-            coreTxtName="${resultDir}/${scenario_alias}_${coreTimeStamp}_${coreName}.txt"
-            gdb $binary_cmd $fcore -ex bt -ex quit > $coreTxtName 2>&1
-            
-            #capture main information of core
-            if [ -s "${coreTxtName}" ];then
-		startLine=`cat $coreTxtName |grep -n "Core was generated by"|awk -F ':' '{print $1}'`
-                sed -i "1,${startLine}d" ${coreTxtName}
-            fi
-
-        done
-        
-        echo "test_error=Y" >> ${resultDir}/summary_info
-
-        #backup db volumes
-        dbVolumeName="${db_name}_volumes"
-        mkdir -p ${dbVolumeName}
-        cp -rf ${CUBRID}/databases/${db_name}/* ${dbVolumeName}
-
-        
-        #backup logs
-        mkdir -p logs
-        cp -rf ${CUBRID}/log/* logs/   
-     fi
-
+     for x in `find $CUBRID ${CTP_HOME} -type f -name "core*"`
+     do
+          isCore=`file $x|grep 'core file'|grep -v grep|wc -l`
+          if [ $isCore -ne 0 ];then
+                let "hasCore=hasCore+1"
+                coreList="$coreList  $x"
```

Comment:
For $coreList, $CUBRID and $CTP_HOME, please consider the case when there is blank character in path directory.